rclpy/test/test_timer.py

```diff
@@ -133,7 +133,8 @@ def func_launch(function, args, message):
 def test_timer_zero_callbacks10hertz():
     func_launch(func_zero_callback, ['0.1'], 'received callbacks when not expected')
 
-
+# TODO(mikaelarguedas) reenable these once timer have consistent behaviour
+# on every platform at high frequency
```

Comment:
You could keep the tests commented in and raise a `SkipTest` exception in their first line for now. That would make them visible in the test results. But no need to do that...